docs/development.md

```diff
@@ -2,6 +2,22 @@
 
 As simple as possible development setup for all three major parts of OpenScanHub (OSH) pipeline. We want to make it compatible with Python 3.6 (the main Python in RHEL 8, supported for its whole lifetime) so we use that specific version inside and outside of the container.
 
+## Quick setup
+
+Execute following commands to quickly try a local OpenScanHub deployment (through podman/docker compose):
+
+```
+git clone git@github.com:openscanhub/openscanhub.git
+cd openscanhub
+git clone git@github.com:release-engineering/kobo.git
+containers/scripts/init-db.sh --full-dev --minimal
+podman exec -it osh-client env OSH_CLIENT_CONFIG_FILE=osh/client/client-local.conf PYTHONPATH=.:kobo python3 osh/client/osh-cli mock-build --config=fedora-37-x86_64 --brew-build units-2.21-5.fc37 --nowait
+```
+
+If the last command is successful, there should be a task accessible at http://localhost:8000/task/1/
+
+Run `podman-compose down` from root directory of the repository to bring down the compose.
```

Comment:
You can also mention the `--clean` option of `deploy.sh` which should remove one traces of OSH from `podman` or `docker`.